source/support.tex

```diff
@@ -5857,3 +5857,215 @@
 corresponding to the signal that caused the invocation of the handler.
 
 \xrefc{7.14}
+
+\rSec1[support.c.headers]{C headers}
+
+\rSec2[support.c.headers.general]{General}
+
+\pnum
+For compatibility with the
+\indextext{library!C standard}%
+C standard library, the \Cpp{} standard library provides
+the \defnx{C headers}{headers!C library} shown in \tref{c.headers}.
+The intended use of these headers is for interoperability only.
+It is possible that \Cpp{} source files need to include
+one of these headers in order to be valid ISO C.
+Source files that are not intended to also be valid ISO C
+should not use any of the C headers.
+
+\begin{note}
+The C headers either have no effect,
+such as \libheaderref{stdbool.h} and \libheaderref{stdalign.h}, or
+otherwise the corresponding header of the form \libheader{cname}
```

Comment:
cname should be a placeholder or at least italicized since it's not real.

Reply:
The `name` in `cname`.

Reply:
fixed